lib/dataset-tools/augmentation/replace_parameters.js

```diff
@@ -106,12 +106,13 @@ class UniformValueList {
 }
 
 class ValueListLoader {
-    constructor(provider, samplingType, rng) {
+    constructor(provider, samplingType, subsetParamSet, rng) {
         this._provider = provider;
 
         this._emptyList = new UniformValueList([]);
         this._cache = new Map;
         this._samplingType = samplingType;
+        this._subsetParamSet = subsetParamSet;
```

Comment:
This option should not be a string, it should be a tuple of numbers. Do the parsing outside.